data/VT/incentives.json

```diff
@@ -230,45 +236,47 @@
     "owner_status": [
       "homeowner"
     ],
-    "low_income": "default-moderate",
     "short_description": {
       "en": "Income-eligible Vermonters can receive $1,000 rebate on a ducted heat pump.",
       "es": "Los residentes de Vermont que cumplan con el nivel de ingresos pueden recibir un reembolso de $1,000 dólares en una bomba de calor con ductos."
-    }
+    },
+    "low_income": "default-moderate"
   },
   {
     "id": "VT-12",
     "authority_type": "utility",
     "authority": "vt-burlington-electric-department",
-    "item": "heat_pump_air_conditioner_heater",
     "payment_methods": [
       "rebate"
     ],
-    "program": "vt_burlingtonElectricDuctedHeatPumpRebateProgram",
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "vt_burlingtonElectricDepartment_burlingtonElectricDepartment-RebateForDuctedHeatPump",
     "amount": {
-      "type": "percent",
-      "number": 0.75,
+      "type": "dollar_amount",
+      "number": 8750,
+      "minimum": 1250,
       "maximum": 8750
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
       "en": "Up to $8,750 on heat pump installation.",
-      "es": "Hasta 8,750 dólares en instalación de bomba de calor."
+      "es": "Hasta 8,750 dólares en instalación de bomba de calor.\n"
```

Comment:
Probably don't want a line break on these? Should the import script do a `.trim()`?